sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogV2Util.scala

```diff
@@ -203,6 +203,14 @@ private[sql] object CatalogV2Util {
               })
           }
 
+        case update: UpdateColumnDefaultValue =>
+          replace(schema, update.fieldNames, field =>
+            if (update.newDefaultValue().nonEmpty) {
```

Comment:
What if the new default value is an empty string? Do we just clear the default value?

Reply:
Oh wait, it seems that will be "\"\""
Then when will the default value get cleared?

Reply:
Good question. The new DEFAULT value string will be non-empty for any DDL commands that set the default value, such as "ALTER TABLE t ALTER COLUMN c SET DEFAULT ..." (this is enforced by the parser). On the other hand, commands that drop the default value such as "ALTER TABLE t ALTER COLUMN c DROP DEFAULT" will set this string to empty. I added this in a comment here to help explain.